src/commands/utils/vatsimData.ts

```diff
@@ -0,0 +1,233 @@
+        if (searchTerms.length <= 1) {
+            const fields: EmbedField[] = [
+                {
+                    name: 'VATSIM details',
+                    value: 'You can find all details about VATSIM [here](https://vatsim.net/).',
+                    inline: false,
+                },
+                {
+                    name: 'Default usage: `.vatsim <callsign query>`',
+                    value: 'Returns information for online Controllers, Observers and ATIS matching the callsign query; or online Pilots with matching callsigns if no Controllers, Observers or ATIS were found.',
+                    inline: false,
+                },
+                {
+                    name: 'Statistics: `.vatsim stats`',
+                    value: 'Returns statics about the number of online Pilots, Controllers and ATIS',
+                    inline: false,
+                },
+                {
+                    name: 'Search online Controllers and ATIS: `.vatsim controllers <callsign query>`',
+                    value: 'Returns information for Controllers, Observers and ATIS matching the callsign query.',
+                    inline: false,
+                },
+                {
+                    name: 'Search online Pilots: `.vatsim pilots <callsign query>`',
+                    value: 'Returns information for Pilots matching the callsign query.',
+                    inline: false,
+                },
+            ];
+            const defaultEmbed = makeEmbed({
+                title: 'VATSIM Data command overview',
+                description: 'This command allows you to retrieve information from VATSIM.',
+                fields,
+            });
+            return msg.channel.send({ embeds: [defaultEmbed] });
+        }
+
+        let commandMode = 'ALL';
+        let notFoundMsg = 'No online VATSIM Controllers, Observers, ATIS or Pilots found matching your search query.';
+        if (subcommandTerms.startsWith(' stats')) {
+            commandMode = 'STATS';
+        } else if (subcommandTerms.startsWith(' controllers')) {
+            searchTerms = subcommandTerms.replace(/\scontrollers\s+/, ' ').split(' ');
+            commandMode = 'CONTROLLERS';
+            notFoundMsg = 'No online VATSIM Controllers, Observers or ATIS found matching your search query.';
+        } else if (subcommandTerms.startsWith(' pilots')) {
+            searchTerms = subcommandTerms.replace(/\spilots\s+/, ' ').split(' ');
+            commandMode = 'PILOTS';
+            notFoundMsg = 'No online VATSIM Pilots found matching your search query.';
+        }
+
+        try {
+            const vatsimData = await fetch(DATA_VATSIM_URL)
+                .then((res) => res.json());
+
+            if (commandMode === 'STATS') {
+                const fields: EmbedField[] = [
+                    {
+                        name: 'Pilots',
+                        value: vatsimData.pilots ? vatsimData.pilots.length : 0,
```

Comment:
Recommend splitting these into const naming them pilotCount or something to make it more clear what the value is

Reply:
Like you did on line 106